public/docs/advanced/truffle-with-metamask.md

```diff
@@ -88,12 +88,12 @@ Click the account icon in the upper-right to create new accounts, the first 10 o
 
 ## Using MetaMask with Truffle Develop
 
-Truffle Develop is a command-line application that runs a temporary blockchain that is also used for testing purposes. It runs on `localhost:9545`.
+Truffle Develop is a command-line application that runs a temporary blockchain that is also used for testing purposes. It runs on `127.0.0.1:9545`.
```

Comment:
Technically this change _is_ more correct, but I think it's more canonical & reads better to leave this one as-is

Reply:
Actually, no - scratch that - don't want to pass the ambiguity on... If anything, we might want to clarify the importance of using `127.0.0.1` instead of `localhost` here...

Reply:
You have the note (see below) within this file already. I disagree with Ben and think this note might be TMI (in both instances. not just this one). We're displaying a big warning meant to prevent a very minor problem. And if people choose to follow our directions rather than go off the rails, they'll be fine without the warning. 

If we must have the warning, can we pare it down? e.g., 

> **Note**: We recommend specifying `127.0.0.1` instead of `localhost` as it does not require a network connection. Specifying `localhost` when offline will cause connection errors.

Reply:
I left it in, but pared it down even further:

> We recommend specifying `127.0.0.1` instead of `localhost` because the address does not require a network connection and so is more suitable for development.